packages/webapp/__tests__/SquadFeedPage.tsx

```diff
@@ -231,15 +231,52 @@ Object.assign(navigator, {
 
 describe('squad header bar', () => {
   it('should show five of the squad member images', async () => {
-    const members = generateMembersResult().sourceMembers.edges;
+    Object.defineProperty(global, 'matchMedia', {
+      writable: true,
+      value: jest.fn().mockImplementation(() => ({
+        matches: true,
+        addListener: jest.fn(),
+        removeListener: jest.fn(),
+      })),
+    });
+    const members = generateMembersResult().sourceMembers.edges.slice(0, 5);
```

Comment:
I would maybe not slice it, that way if someone ever removes the splice on component level we get a failing test?

Reply:
If someone removes the splice from component then the tests should fail, I think that's the whole point of being exact. WDYT?

Reply:
Yeah but if the test always splice 5 wont it limit the case for when there would be mote then 5?
Maybe misread the test

Reply:
But there is no such case since `SquadMemberShortList` always shows 3 or 5, and we have tests for both?

Reply:
Ah yeah is what i mean
I only added the 3-5 constraint now, so think it be valid to ensure nobody changes the limit accidentally if that makes sense.

Reply:
Yeah, thats why the initial test failed so I added both to ensure we can track it 🤝